Lib/fontTools/help.py

```diff
@@ -0,0 +1,51 @@
+import pkgutil
+import sys
+import fontTools
+import importlib
+import os
+from pathlib import Path
+
+def _discover_path_importables(pkg_pth, pkg_name):
+    """Yield all importables under a given path and package."""
+    for dir_path, _d, file_names in os.walk(pkg_pth):
+        pkg_dir_path = Path(dir_path)
+
+        if pkg_dir_path.parts[-1] == "__pycache__":
+            continue
+
+        if all(Path(_).suffix != ".py" for _ in file_names):
+            continue
+
+        rel_pt = pkg_dir_path.relative_to(pkg_pth)
+        pkg_pref = ".".join((pkg_name,) + rel_pt.parts)
+        yield from (
+            pkg_path
+            for _, pkg_path, _ in pkgutil.walk_packages(
+                (str(pkg_dir_path),), prefix=f"{pkg_pref}.",
+            )
+        )
+
+
+def main():
+    """Show this help"""
+    path = fontTools.__path__
+    descriptions = {}
+    for pkg in sorted(
+        set(_discover_path_importables(fontTools.__path__[0], "fontTools"))
+    ):
+        try:
+            description = __import__(
+                pkg, globals(), locals(), ["main"]
+            ).main.__doc__
+            if description:
+                pkg = pkg[10:].replace(".__main__", "")
```

Comment:
what's 10? is it `len("fontTools.")`? I'd just remove it, you know it's not going to affect the result of this specific replace

Reply:
It will affect the result. The package name is fully qualified (`fontTools.feaLib.__main__`) and we want to turn it into a verb `feaLib` for display in the list e.g.

```
fonttools feaLib Add features from a feature file (.fea) into a OTF font
```

We could change the first "." to a space but that's more obscure. :-)

Reply:
ok I see, thanks for the clarification. I'd still replace that magic constant with `len("fontTools.")` for the sake of readability